prometheus2/tests/test.bats

```diff
@@ -1,22 +1,17 @@
-source "${BATS_TEST_DIRNAME}/../plan.sh"
-
-@test "Command is on path" {
-  [ "$(command -v prometheus)" ]
-  [ "$(command -v promtool)" ]
-}
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
 
 @test "Version matches" {
-  result="$(prometheus --version 2>&1 | head -n 1 | awk '{print $3}')"
-  [ "$result" = "${pkg_version}" ]
+  result="$(hab pkg exec ${TEST_PKG_IDENT} prometheus --version 2>&1 | head -n 1 | awk '{print $3}')"
+  [ "$result" = "${TEST_PKG_VERSION}" ]
 }
 
 @test "Help command" {
-  run prometheus --help
+  run hab pkg exec ${TEST_PKG_IDENT} prometheus --help
   [ $status -eq 0 ]
 }
 
 @test "Service is running" {
-  [ "$(hab svc status | grep "prometheus2\.default" | awk '{print $4}' | grep up)" ]
+  [ "$(hab svc status | grep "prometheus\.default" | awk '{print $4}' | grep up)" ]
```

Comment:
i think this needs to be `prometheus2\.default` still

Reply:
Fixed